src/nos/core/notifications.py

```diff
@@ -126,3 +133,58 @@ def formatted(self, **kwargs) -> "Notification":
     "File upload completed",
     "The file {file_id} has been successfully uploaded.",
 )
+
+ALL_IVAS_INVALIDATED_TO_USER = Notification(
+    "IVA Invalidation",
```

Comment:
On the user facing pages of the data portal UI, we avoid the technical term "IVA", but call it "contact address (for account verification)". We should use the same terminology in the notifications sent to users.